components/VM_Arm/src/main.c

```diff
@@ -927,6 +941,28 @@ static int load_vm(vm_t *vm, const char *kernel_name, const char *dtb_name, cons
     return 0;
 }
 
+static unsigned long parse_camkes_vm_entry_addr(void)
+{
+    unsigned long addr = strtoul(vm_address_config.kernel_entry_addr, NULL, 0);
```

Comment:
What if `kernel_entry_addr` was an empty string by default and you used `if (!strlen(vm_address_config.kernel_entry_addr))` to branch to computing entry address based on architecture? No need to have yet another `#define`.

Reply:
That was the behavior I had originally. After discussions with @axel-h we implemented it in a way that forces the user to define the value.

However, if community feedback wants implemented in a different manner, I can implement in that way.

Thoughts @kent-mcleod @wom-bat @lsf37?